app/src/main/java/com/zulip/android/ZulipApp.java

```diff
@@ -142,10 +142,7 @@ public String getServerURI() {
         if (getEmail().equals("iago@zulip.com")) {
             return "http://10.0.2.2:9991/api/";
         }
-        return settings.getString(
-            "server_url",
-            "https://api.zulip.com/"
-        );
+        return settings.getString("server_url", "https://api.zulip.com/");
```

Comment:
this is just line-wrapping right?  LGTM but maybe commit message should make that clear.